CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing Guidelines
+
+Thanks for interest in contribution to `colin`.
+
+The following is a set of guidelines for contributing to `colin`. These are not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How to contribute to colin
+
+### Reporting Bugs
+Before creating bug reports, please check a [list of known issues](https://github.com/user-cont/colin/issues) to see
+if the problem has already been reported.
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/user-cont/colin/issues/new).
+Be sure to include a **descriptive title, clear description and a package version** and please include **as many details as possible** to help maintainers reproduce
+the issue and resolve it faster.
+If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/).
```

Comment:
This is not the first time you're talking about github issues in this document, so I'd either move the link into `Reporting Bugs` section or remove completely (Is there anybody who doesn't know what's github issue ?).

Reply:
Removing it completely.